### ISSUE TITLE
Overwrite ContextMenuTrigger for some child elements

### ISSUE THREAD

**Issue by quajak** [stavroskasidis/BlazorContextMenu#81]
**Describe the bug**
When creating a second ContextMenuTrigger, within a ContextMenuTrigger, and an element within the inner trigger is clicked both are shown and not only the inner one.

**To Reproduce**
Run the following code:
```
<ContextMenuTrigger MenuId="menu">
    <div>
        <p>Test</p>
        <ContextMenuTrigger MenuId="special">
            <div>Special</div>
        </ContextMenuTrigger>
    </div>

</ContextMenuTrigger>

<ContextMenu Id="menu">
    <Item >Item 1</Item>
    <Item> Item 2</Item>
    <Item> Item 3</Item>
</ContextMenu>

<ContextMenu Id="special">
    <Item>Special</Item>
</ContextMenu>
```
And click on special.

**Expected behavior**
It should only show one menu with the item special.

**Screenshots**
![image](https://user-images.githubusercontent.com/8559822/92392781-1cf01e00-f11f-11ea-8531-2cfa6bf46828.png)

**Comment by stavroskasidis** [comment]
Thank you for the reproduction steps. I will work on this asap

**Comment by stavroskasidis** [comment]
Fixed in [1.7](https://www.nuget.org/packages/Blazor.ContextMenu/1.7.0)

**Comment by quajak** [comment]
Thanks!
